Samples/CommandBinding/MauiEdit.UnitTest/UiControllerTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using MauiEdit.Services;
+using MauiEdit.ViewModels;
+
+namespace MauiEdit.UnitTest;
+
+public class MainFormUiControllerTest {
+
+    [Fact]
+    public async Task MainFormNewDocumentTest() {
+        var service = SimpleTestServiceProvider.GetInstance();
+        var dialogService = (SimpleUnitTestDialogService?)service.GetService(typeof(IDialogService));
+        int dialogState = 0;
+
+        Assert.NotNull(dialogService);
+
+        MainFormController mainFormController = new(service)
+        {
+            // We clear the main document.
+            TextDocument = string.Empty
+        };
+
+        // We assert, that the New command is disabled, and cannot be called.
```

Comment:
Are these your comments or part of the template? Should avoid "We" and there is an unnecessary comma.

Reply:
Mine. I'll correct that.